### PR TITLE
task/sync-master-develop-after-release

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard26**
## Summary

This pull request brings `develop` up-to-date with some changes that did make it into `master` but not `develop`.

### CODE REVIEW

File: packages/app/next.config.js [hunk 1]
```diff
@@ -5,12 +5,7 @@ const LodashModuleReplacementPlugin = require('lodash-webpack-plugin');
 const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true',
 });
-const withTranspileModules = require('next-transpile-modules')([
-  'd3-geo',
-  'd3-array',
-  'globby',
-  'internmap',
-]);
+const withTranspileModules = require('next-transpile-modules')(['d3-geo', 'd3-array', 'globby', 'internmap']);
```
Comment (VWSCoronaDashboard26) [inline packages/app/next.config.js:8]: This file has been formatted. Line 57 has been changes from `size` to `named`.

File: packages/app/src/domain/layout/components/gm-combo-box.tsx [hunk 2]
```diff
@@ -21,11 +21,7 @@ export function GmComboBox(props: GmComboBoxProps) {
       placeholder={commonTexts.common.zoekveld_placeholder_gemeente}
       options={gmData}
       onSelect={({ gemcode }) => {
-        router.push(
-          typeof getLink === 'function'
-            ? getLink(gemcode)
-            : reverseRouter.gm.index(gemcode)
-        );
+        router.push(typeof getLink === 'function' ? getLink(gemcode) : reverseRouter.gm.index(gemcode));
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/domain/layout/components/gm-combo-box.tsx:24]: This file has been formatted. A `sorter` prop has been removed.

File: packages/app/src/domain/layout/components/vr-combo-box.tsx [hunk 3]
```diff
@@ -20,13 +20,7 @@ export function VrComboBox(props: VrComboBoxProps) {
...
       placeholder={commonTexts.common.zoekveld_placeholder_regio}
       options={vrData}
-      onSelect={(region) =>
-        router.push(
-          typeof getLink === 'function'
-            ? getLink(region.code)
-            : reverseRouter.vr.index(region.code)
-        )
-      }
+      onSelect={(region) => router.push(typeof getLink === 'function' ? getLink(region.code) : reverseRouter.vr.index(region.code))}
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/domain/layout/components/vr-combo-box.tsx:23]: This file has been formatted. A `sorter` prop has been removed.

File: packages/app/src/components/combo-box/combo-box.tsx [hunk 4]
```diff
@@ -1,10 +1,4 @@
-import {
-  Combobox,
-  ComboboxInput,
-  ComboboxList,
-  ComboboxOption,
-  ComboboxPopover,
-} from '@reach/combobox';
+import { Combobox, ComboboxInput, ComboboxList, ComboboxOption, ComboboxPopover } from '@reach/combobox';
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/components/combo-box/combo-box.tsx:1]: This file has been formatted. A `sorter` prop has been removed.

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Review by VWSCoronaDashboard25 — APPROVED** [review]

**Review by VWSCoronaDashboard25 — APPROVED** [review]

**Pull request merged**